tools/go.mod

```diff
@@ -1,7 +1,8 @@
 module github.com/csi-addons/kubernetes-csi-addons/tools
 
-go 1.22.0
-toolchain go1.22.8
+go 1.22.7
+
+toolchain go1.23.0
```

Comment:
@Madhu-1 , was there some issue with `toolchain` updates?

Reply:
i think this is not required, i dont see any dependency update that required 1.23